### PR TITLE
PE-2581: Localizely Integration

### PR DESCRIPTION
**Pull request by matibat**
**Changes**
- Adds the YAML configuration for the Integration with Localizely

--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/43ipoo0fel860
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/193g6deo29bno

### CODE REVIEW

File: localizely.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+config_version: 1.0 # Only 1.0 available
+project_id: 062e9d52-7961-4b8c-9079-a85bca6ed73a # Your project ID from: https://app.localizely.com/projects
+file_type: flutter_arb
+branch: dev
```
Comment (matibat) [inline localizely.yaml:4]: We are gonna read from and commit to the `dev` branch

File: localizely.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+upload:
+  files: # List of files for upload to Localizely. Usually it is just one file used for the main locale
+    - file: lib/l10n/app_en.arb
```
Comment (matibat) [inline localizely.yaml:8]: The only source of truth for Localizely will be the English translations

File: localizely.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      locale_code: en
+  params:
+    overwrite: true # Optional, default: false. If the translation in a given language should be overwritten with modified translation from uploading file.
+    reviewed: false # Optional, default: false. If uploading translations, that are added, should be marked as Reviewed.
```
Comment (matibat) [inline localizely.yaml:12]: Any new String Key will be overwritten, and updated ones will be marked as non-reviewed

File: localizely.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    tag_added: # Optional. List of tags to add to new translations from uploading file.
+      - added
+    tag_removed: # Optional. List of tags to add to removed translations from uploading file.
+      - removed
+    tag_updated: # Optional. List of tags to add to updated translations from uploading file.
+      - updated
```
Comment (matibat) [inline localizely.yaml:18]: This is the first time we are using tags for the translations. We're gonna use it to track the changes made to the Strings

File: localizely.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    - file: lib/l10n/app_en.arb
+      locale_code: en
+    - file: lib/l10n/app_es.arb
+      locale_code: es
+    - file: lib/l10n/app_ja.arb
+      locale_code: ja
+    - file: lib/l10n/app_zh-HK.arb
+      locale_code: zh-HK
+    - file: lib/l10n/app_zh.arb
+      locale_code: zh
```
Comment (matibat) [inline localizely.yaml:31]: We track here all of the localization files, linked to the path relative to the root of the Repository. The Integration will create PRs for these whenever there are changes

File: localizely.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  params:
+    export_empty_as: skip # Optional, default: empty. How you would like empty translations to be exported. Allowed values are `empty` to keep empty, `main` to replace with the main language value, or `skip` to omit.
```
Comment (matibat) [inline localizely.yaml:33]: If there's a resource file for a new language which is empty, then it won't be exported.

**Review by matibat — COMMENTED** [review]
\* self-review checkpoint *

**Review by karlprieb — APPROVED** [review]

**Pull request merged**